tests/test_consumer.py

```diff
@@ -132,6 +132,15 @@ def test_create_consumer_no_running_loop(self):
             loop.run_until_complete(consumer.stop())
             loop.close()
 
+    @run_until_complete
+    async def test_create_consumer_with_listener(self):
+        listener = StubRebalanceListener()
```

Comment:
There are 2 tests here, it would be better to separate them. 
And the both test check validation only. But we also have to check that custom listener is actually used, can we do that too?